Assets/AppCenter/link.xml

```diff
@@ -11,4 +11,13 @@
     <assembly fullname="Microsoft.AppCenter.Crashes" preserve="all" ignoreIfMissing="1"/>
     <assembly fullname="Microsoft.AppCenter.Push" preserve="all" ignoreIfMissing="1"/>
     <assembly fullname="SQLite-net" preserve="all" ignoreIfMissing="1"/>
+    <!-- Name 'Assembly-CSharp' could be changed through 'Assembly Definitions'. In this case we need to change full name here. -->
```

Comment:
not necessary "we". Just use passive voice

Reply:
Fixed